File: javaPOO/POOAula09/src/pooaula09/Pessoa.java
package pooaula09;
public class Pessoa {
    private String pessoa, sexo;
    private int idade;

    public Pessoa(String pessoa, String sexo, int idade) {
        this.pessoa = pessoa;
        this.sexo = sexo;
        this.idade = idade;
    }
    
    
    public String getPessoa() {
        return pessoa;
    }

    public void setPessoa(String pessoa) {
        this.pessoa = pessoa;
    }

    public String getSexo() {
        return sexo;
    }

    public void setSexo(String sexo) {
        this.sexo = sexo;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public void fazerAniver(){
        this.setIdade(this.getIdade()+1);
    }
}

File: phpPOO/POOAula14/Pessoa.php
<?php

abstract class Pessoa {

    protected $nome, $sexo;
    protected $idade;
    protected $exp;
    
    public function __construct($nome, $idade,$sexo) {
        $this->nome = $nome;
        $this->sexo = $sexo;
        $this->idade = $idade;
        $this->exp = 0;
    }

    
    protected function ganharXp($n) {
        $this->exp += $n;
    }

    
    public function getNome() {
        return $this->nome;
    }

    public function getSexo() {
        return $this->sexo;
    }

    public function getIdade() {
        return $this->idade;
    }

    public function getExp() {
        return $this->exp;
    }

    public function setNome($nome): void {
        $this->nome = $nome;
    }

    public function setSexo($sexo): void {
        $this->sexo = $sexo;
    }

    public function setIdade($idade): void {
        $this->idade = $idade;
    }

    public function setExp($exp): void {
        $this->exp = $exp;
    }

}
